{"status": "broken", "uuid": "8048a2f9-f1e3-405d-9fee-5102dd9c88cd", "labels": [{"name": "story", "value": "角色名称修改接口"}, {"name": "feature", "value": "角色管理"}, {"name": "tag", "value": "@pytest.mark.usefixtures('userId')"}, {"name": "suite", "value": "test_set"}, {"name": "subSuite", "value": "TestRole"}, {"name": "parentSuite", "value": "testcase"}, {"name": "host", "value": "DESKTOP-M0PGOTF"}, {"name": "thread", "value": "13804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "testcase.test_set"}], "stop": 1629252153692, "testCaseId": "db372feb7f99229b92b3d0b098808040", "historyId": "bbbbfc1d60e74d9125435c7fdf23f5ff", "start": 1629252153692, "fullName": "testcase.test_set.TestRole#test_role_update", "statusDetails": {"message": "TypeError: 'NoneType' object has no attribute '__getitem__'", "trace": "@pytest.fixture(scope='session')\n    def userId():\n>       userId = Login().yt_login('易通')['data']['userId']\nE       TypeError: 'NoneType' object has no attribute '__getitem__'\n\ntestcase\\conftest.py:23: TypeError"}, "name": "test_role_update"}